Introduced in Android 9, the adaptive brightness feature raises or lower the brightness of the screen depending on the light in the current environment.

For some reasons, developers may disable this feature programmatically, setting the field `WindowManager.LayoutParams#screenBrightness` with the constant `BRIGHTNESS_OVERRIDE_FULL`.

This feature was introduced to improve battery life, be careful when deactivating it.

## Noncompliant Code Example

```java
getWindow().getAttributes().screenBrightness = BRIGHTNESS_OVERRIDE_FULL;
```
