In contrast to classic Bluetooth, Bluetooth Low Energy (BLE) is designed to provide significantly lower power consumption. Its purpose is to save energy on both paired devices but very few developers are aware of this alternative API.

From the Android client side, it means append `android.bluetooth.le.\*` imports to `android.bluetooth.*` imports in order to benefits from low-energy features.

## Noncompliant Code Example

```java
import android.bluetooth.*
```

## Compliant Solution

```java
import android.bluetooth.le.*
```
