Score ecoCode: 2sur5

Turning on the torch mode programmatically must absolutely be avoided because the flashlight is one of the most energy-intensive component.

## Noncompliant Code Example

```swift
AVCaptureTorchMode.on
```

or

```swift
    setTorchModeOn(level: Float)
```

or

```swift
torchMode = .on
```
