During a voice interaction session, `VoiceInteractionSession#setKeepAwake(boolean keepAwake)` allows to decide whether it will keep the device awake while it is running a voice activity.

By default, the system holds a wake lock for it while in this state, so that it can work even if the screen is off. Setting this to `false` removes that wake lock, allowing the CPU to go to sleep and hence does not let this continue to drain the battery.

## Noncompliant Code Example

```java
VoiceInteractionSession voiceSession = new VoiceInteractionSession(this);
```

or

```java
VoiceInteractionSession voiceSession = new VoiceInteractionSession(this);
voiceSession.setKeepAwake(true);
```

## Compliant Code Example

```java
VoiceInteractionSession voiceSession = new VoiceInteractionSession(this);
voiceSession.setKeepAwake(false);
```
