With Bluetooth Low Energy technology (see BLE API smell), a Bluetooth Smart Ready device (the master) will establish a link with a Bluetooth Smart device (the slave).

Most often, the slave is a GATT server and the master is a GATT client. GATT capable devices can be discovered using BLE scan process.

Invoking `BluetoothGatt#requestConnectionPriority(int connectionPriority)` with the value `CONNECTION_PRIORITY_LOW_POWER`, is recommended to reduce power consumption.

## Noncompliant Code Example

```java
BluetoothGatt gatt = new BluetoothGatt();
gatt.requestConnectionPriority(CONNECTION_PRIORITY_HIGH);
```

## Compliant Code Example

```java
BluetoothGatt gatt = new BluetoothGatt();
gatt.requestConnectionPriority(CONNECTION_PRIORITY_LOW_POWER);
```
