Score ecoCode: 2sur5

Displaying dark colors is particularly beneficial for mobile devices with (AM)OLED screens.

Consequently, custom resources like bright colors values and bitmap images with too high luminance should be avoided.

## Noncompliant Code Example

```xml
<TextView
    android:layout_width="match_parent"
    android:layout_height="match_parent"
    android:foreground="#FFFFFF"
    android:foregroundTint="#FFFFFF">
</TextView>
```
